tests/test_integrity.py

```diff
@@ -539,6 +539,18 @@ def test_special_characters(db):
                     assert all(check), f'{char} in {table_name}'
 
 
+def test_database_views(db):
+    # Tests to verify views exist and work as intended
+
+    # Views do not exist as attributes to db so db.ViewName does not work
+    # TODO: Figure out other ways to refer to it in db.metadata info
+    t = db.query(ParallaxView).table()
```

Comment:
Should also test the other view for codecov no?

Reply:
We can, but bear in mind that all the database schema is run as part of creation so it is technically covered even if no explicit test exists.